platform/firefox/manifest.json

```diff
@@ -28,9 +28,18 @@
         "nativeMessaging",
         "webRequest",
         "webRequestBlocking",
-        "storage"
+        "storage",
+        "activeTab"
     ],
     "host_permissions": [
-        "https://login.microsoftonline.com/*"
+        "https://login.microsoftonline.com/*",
```

Comment:
We will stick to just `https://login.microsoftonline.com/*`, as only this is needed for Thunderbird. All other domains can either be enabled via managed settings (policy), or by the end user.